### ISSUE TITLE
URNs as schema ids

### ISSUE THREAD

**Issue by sjahreis** [pwall567/json-kotlin-schema-codegen#7]
Hi there,

I really like your project. But I’m having some issues with my schemas.

Are you planning to support URNs as schema ids? Currently this results in the  class name `Urn`.

Example:
```
{
    "id": "urn:jsonschema:com:example:Person",
…
}
```

Thanks!

**Comment by pwall567** [comment]
Hi - thanks for the feedback.

I haven't previously had requests to handle URNs, but in response to earlier suggestions I have been considering adding a configuration file mechanism, which would allow something like the following (the format is undecided):
```
{
  "classNames": [
    { "id": "urn:jsonschema:com:example:Person", "name": "Person" }
  ]
}
```
Other sections of the same file could be used to specify custom classes etc.

This is just in the planning stage (and feedback is welcome) but in the meantime I have made a minor change to the code that selects the generated class name, skipping all characters before the last ':' in the URI.  This should make the case that you describe above use the name "Person" (and I hope it doesn't break any existing uses).

The new version 0.63 should be available very shortly in Maven Central - I hope this helps.

**Comment by sjahreis** [comment]
Thanks for your fast solution!

The idea with the config file you mentioned sounds great.

**Comment by pwall567** [comment]
Hi @sjahreis - I'm not sure whether you're still following this issue, but the initial implementation of the config file that I mentioned earlier is now available in version 0.66.  The documentation is at  [CONFIG.md](https://github.com/pwall567/json-kotlin-schema-codegen/blob/main/CONFIG.md) and the section on generated class name selection is at [classNames](https://github.com/pwall567/json-kotlin-schema-codegen/blob/main/CONFIG.md#classnames).

I hope this is useful - all feedback welcome.

**Comment by sjahreis** [comment]
Awesome! Thanks!
